src/Stratis.Bitcoin.Features.Dns.Tests/GivenAWhitelistManager.cs

```diff
@@ -78,20 +79,22 @@ public void WhenConstructorCalled_AndDnsServerIsNull_ThenArgumentNullExceptionIs
 
         [Fact]
         [Trait("DNS", "UnitTest")]
-        public void WhenConstructorCalled_AndNodeSettingsAreNull_ThenArgumentNullExceptionIsThrown()
+        public void WhenConstructorCalled_AndConnectionSettingsAreNull_ThenArgumentNullExceptionIsThrown()
         {
             // Arrange.
             IDateTimeProvider dateTimeProvider = new Mock<IDateTimeProvider>().Object;
             ILoggerFactory loggerFactory = new Mock<ILoggerFactory>().Object;
             IPeerAddressManager peerAddressManager = new Mock<IPeerAddressManager>().Object;
             IDnsServer dnsServer = new Mock<IDnsServer>().Object;
+            DnsSettings dnsSettings = new Mock<DnsSettings>().Object;
+            dnsSettings.DnsHostName = "stratis.test.com";
 
-            Action a = () => { new WhitelistManager(dateTimeProvider, loggerFactory, peerAddressManager, dnsServer, null, null); };
+            Action a = () => { new WhitelistManager(dateTimeProvider, loggerFactory, peerAddressManager, dnsServer, null, dnsSettings); };
 
             // Act and Assert.
-            a.ShouldThrow<ArgumentNullException>().Which.Message.Should().Contain("nodeSettings");
+            a.ShouldThrow<ArgumentNullException>().Which.Message.Should().Contain("connectionSettings");
         }
-
+/*
         [Fact]
         [Trait("DNS", "UnitTest")]
         public void WhenConstructorCalled_AndNodeSettingsDnsHostNameIsNull_ThenArgumentNullExceptionIsThrown()
```

Comment:
If this test is never going to be applicable again, we can just remove it altogether 👍

Reply:
Done.